docs/output.md

```diff
@@ -145,6 +145,21 @@ The resulting FASTQ files will only be present in your results directory if you
 
   [BWA](https://bio-bwa.sourceforge.net/bwa.shtml) is a software package for mapping low-divergent sequences against a large reference genome, such as the human genome. BWA-backtrack (a.k.a `bwa aln`) is designed for Illumina sequence reads up to 100bp, while BWA-MEM (`bwa mem`) is optimised for longer sequences ranged from 70bp to 1Mbp and split alignment.
 
+#### BOWTIE 2
```

Comment:
```suggestion
#### Bowtie2 2
```

Reply:
```suggestion
#### BOWTIE2
```
surely you mean this?

Reply:
Partly, also lower case - we generally want to refer to the tool by how it's referred to in the literature (so the paper says Bowtie2 rather than BOWTIE2 or bowtie2)

Reply:
The Nature Methods paper actually says Bowtie 2